new/LIT7237QeneFequrana.xml

```diff
@@ -0,0 +1,77 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ፍቁራነ፡ ኮኑ፡ ወኢተሴሰዩ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Fǝqurāna konu wa-ʾitasessayu (Qǝne of the mawaddǝs-kʷǝllǝkǝmu-type)</title>
+                <title xml:lang="en" corresp="#t1">They were happy even though they were not fed…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Neither type nor author of the qǝne are indicated in the manuscript. According to the classification by 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl> a qǝne of the 
+                    mawaddǝs-kʷǝllǝkǝmu-type.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Kwellekomu"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-01-12">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/>, fol. 56ra-b. Division into verses is indicated with ። . 
+                    The end of the poem is indicated with ። ። .</note>
+                <ab>
+                    <l n="1">ፍቁራነ፡ ኮኑ፡ ወኢተሴሰዩ፡ <add place="above">ጊዜ፡ ሞቶሙ፡ ረድኤት፡ ወ</add>ጊዜ፡ ሕይወቶሙ፡ ሣህል<supplied reason="omitted">።</supplied></l> 
```

Comment:
The place of the addition appears ok for me.

Reply:
Me too! :)